### ISSUE TITLE
Image | NanoPi Fire3

### ISSUE THREAD

**Issue by Fourdee** [MichaIng/DietPi#1926]
Many thanks to FriendlyElec for sending us dev sample boards!

> http://wiki.friendlyarm.com/wiki/index.php/NanoPi_Fire3#Essentials_You_Need

**Comment by Fourdee** [comment]
Notes:
- Using pre-image with Debian RootFS replace:
s5p6818-lubuntu-desktop-xenial-4.4-armhf-YYYYMMDD.img.zip
🈴 No terminal HDMI output. Only the FriendlyARM boot logo is displayed. sets 720p but no text.

**Comment by Fourdee** [comment]
🈴 Idle CPU.  Provided heatsink and thermal pad installed:
```
Temp         |     65'c : 149'f | Running hot, not recommended.
```

Hits 80'c (thermal throttling) during our 10 second CPU benchmark.

![image](https://user-images.githubusercontent.com/7077026/42774584-f89bf42a-8929-11e8-943d-ce088e1b2eeb.png)

🈴 Appears CPU voltage scaling is fixed at max voltage. Scaling CPU has no effect on CPU temps:
```
 ─────────────────────────────────────────────────────
 DietPi CPU Info
 Use dietpi-config to change CPU / performance options
 ─────────────────────────────────────────────────────
 Architecture |     armv7l
 Temp         |     66'c : 150'f | Running hot, not recommended.
 Governor     |     ondemand
 Throttle up  |     50% CPU usage

                 Current Freq    Min Freq   Max Freq
 CPU0         |      500 MHz      400 MHz    500 MHz
 CPU1         |      500 MHz      400 MHz    500 MHz
 CPU2         |      500 MHz      400 MHz    500 MHz
 CPU3         |      500 MHz      400 MHz    500 MHz
 CPU4         |      500 MHz      400 MHz    500 MHz
 CPU5         |      500 MHz      400 MHz    500 MHz
 CPU6         |      500 MHz      400 MHz    500 MHz
 CPU7         |      500 MHz      400 MHz    500 MHz
```

**Comment by Fourdee** [comment]
Email sent to FriendlyARM regarding these issues. Will put this on hold until we get a response.

**Comment by Fourdee** [comment]
#### Ok, regarding the HDMI no terminal output issue:
Appears tty1 is disabled in kernel/bootloader. tty2 is functional and can be switched with:
- CTRL+ALT+F2

**Comment by Fourdee** [comment]
Completed, image live.
https://twitter.com/DietPi_/status/1019346392135938048

NB: CPU temp readout fixed in v6.12 update.
